src/app/researcher/study/request/[memberSlug]/study-proposal-form.tsx

```diff
@@ -31,95 +39,81 @@ export const StudyProposalForm: FC<{
     const { user } = useUser()
 
     return (
-        <Paper p="md">
-            <Stack gap="lg">
-                <Title order={4}>Study Proposal</Title>
-                <Divider />
-                <Text>
-                    This section is here to help you submit your study proposal. Consider providing as much detail as
-                    possible to ensure the Reviewer has all the information needed to make an informed decision.
-                </Text>
+        <Paper p="xl">
+            <Title order={4}>Study Proposal</Title>
+            <Divider my="md" />
+            <Text mb="md">
+                This section is here to help you submit your study proposal. Consider providing as much detail as
+                possible to ensure the Reviewer has all the information needed to make an informed decision.
+            </Text>
```

Comment:
Could potentially <Stack>-ify this with a gap, rather than having the margins explicitly, but this is just me nitpicking